### ISSUE TITLE
Multiple "Task log" for concurrent builds

### ISSUE THREAD

**Issue by princejohnsantillan** [laravel-shift/workbench-issues#4]
**Desktop:**
 - OS: Mac
 - OS Version: Big Sur 11.4
 - App Version: 0.2.0

**Describe the bug**
Spinning cog on Cleaning Up and Generating Report does not stop even though the docker process already finished and closed.

**To Reproduce**
1. Run common-helpers
2. Run global-facades (don't wait for step one to finish)
3. Run other tasks (don't wait for other tasks to finish)

![Screen Shot 2021-07-01 at 9 02 47 AM](https://user-images.githubusercontent.com/60916966/124049477-23071080-da4b-11eb-9dbf-472fdf9ca83e.png)

**Comment by jasonmccreary** [comment]
@princejohnsantillan, when you say "don't wait for step one to finish" are you running another task before the first "build" completed?

**Comment by princejohnsantillan** [comment]
@jasonmccreary yes I am. I do see another container being spawned in Docker every time I run another task. So I'm assuming they run in parallel safely.

**Comment by jasonmccreary** [comment]
Probably true, however, the "Task Log" did not account for this. 😅 

I'll turn this into a feature so you can spawn multiple at once. Until then, may want to run them one at a time. 😉

**Comment by princejohnsantillan** [comment]
They actually did run in parallel and completed as expected. It's just the logs are off. 😅
Maybe there will be conflicting tasks that shouldn't run in parallel else there'll be some race conditions to consider. 
But if I think the tasks are different I just run them together.

**Comment by jasonmccreary** [comment]
@princejohnsantillan, good. I still think that'd be a cool feature though. No reason we couldn't create multiple "Task Log" tabs in the console.

But yeah, kind of up to the dev if they wanted to run so many at once.

**Comment by jasonmccreary** [comment]
We recently released v1.0. However, this didn't make the cut. I'm going to keep it in mind, but close this for now.
